### PR TITLE
feat(footer) update

### PR DESCRIPTION
**Pull request by adamjkuhn**
### Summary

Update footer for consistency with konghq.com

### Full changelog

* strip newsletter bar from internal pages and place in footer column
* adjust column widths & breakpoints
* update footer styles for consistency
* add reCaptcha to footer form

- [x] [Commit message & atomicity](https://github.com/Kong/docs.konghq.com/blob/master/CONTRIBUTING.md#commit-atomicity) checked
- [x] Documentation 
- [x] Spellchecked my updates
- [x] Ready to be merged

### CODE REVIEW

File: app/_assets/stylesheets/footer.less [hunk 1]
```diff
@@ -42,15 +41,138 @@
...
+              min-width: 200px;
+              margin-bottom: 0;
+            }
+          }
+        }
+
         @media (min-width: @grid-width-sm) {
           .flex-col(1, 1, 3);
+          &.logo-wrapper {
+            flex-basis: 33.33333334%;
```
Comment (h3xar0n) [inline app/_assets/stylesheets/footer.less]: (not a blocker)

```suggestion
            flex-basis: 33.33333333%;
```

File: app/_includes/footer.html [hunk 2]
```diff
@@ -1,57 +1,121 @@
...
-            <li><a href="https://konghq.com/api-dev-portal/">Dev Portal</a></li>
-            <li><a href="https://konghq.com/api-analytics/">API Analytics</a></li>
-            <li><a href="https://konghq.com/kong-enterprise-edition/">Enterprise</a></li>
+            <li>
+              <a href="https://support.konghq.com" target="_blank"
+                >Enterprise Support</a
+              >
+            </li>
+            <li>
+              <a href="https://docs.konghq.com" target="_blank"
```
Comment (h3xar0n) [inline app/_includes/footer.html]: Is there a reason these two open new tabs, but the old ones don't?

File: app/_includes/footer.html [hunk 3]
```diff
@@ -1,57 +1,121 @@
...
       </li>
       <li>
         <nav>
-          <h4>Legal</h4>
+          <h5>Open Source</h5>
           <ul>
-            <li><a href="https://konghq.com/terms/">Terms of Service</a></li>
-            <li><a href="https://konghq.com/privacy/">Privacy Policy</a></li>
+            <li><a href="http://konghq.com/install/">Install</a></li>
+            <li><a target="blank" href="https://github.com/Kong/">GitHub</a></li>
```
Comment (h3xar0n) [inline app/_includes/footer.html]:
```suggestion
            <li><a target="_blank" href="https://github.com/Kong/">GitHub</a></li>
```

File: app/_includes/footer.html (hunk 2 above)
Comment (adamjkuhn) [reply app/_includes/footer.html]: yeah the reason is it's a clone of konghq.com on which this is the intended action- good catch

File: app/_includes/footer.html [hunk 4]
```diff
@@ -61,19 +125,9 @@ <h4>Legal</h4>
...
-          <a href="https://www.facebook.com/konghq/" title="Facebook"><i aria-label="Facebook" class="fa fa-facebook-official" aria-hidden="true"></i></a></div>
-        <div class="social-link">
-          <a href="https://www.meetup.com/topics/kong/all/" title="Meetup"><img aria-label="Meetup" alt="Meetup" src="/assets/images/footer/icn-meetup-black.svg"></a>
-        </div>
-        <div class="social-link">
-          <a href="https://twitter.com/thekonginc" title="Twitter"><i aria-label="Twitter" class="fa fa-twitter" aria-hidden="true"></i></a>
-        </div>
+        <span class="mashape-footer-content"
+          >&copy; Kong Inc. {{ site.time | date: '%Y' }} &nbsp; <a href="http://konghq.com/terms/">Terms</a><b>•</b><a href="http://konghq.com/privacy/">Privacy</a></span
+        >
```
Comment (h3xar0n) [inline app/_includes/footer.html]: The closing `>` on 129 and 130 seem out of place; wish I could suggest multiple lines to save you time.

**Pull request merged**